src/test/kotlin/org/springboard/admin/AdminServiceTest.kt

```diff
@@ -0,0 +1,85 @@
+package org.springboard.admin
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springboard.auth.AuthAuthority
+import org.springboard.user.UserCreationRequest
+import org.springboard.user.UserCreationResult
+import org.springboard.user.UserServiceImpl
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import java.util.*
+
+@ExtendWith(SpringExtension::class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+class AdminServiceTest {
```

Comment:
Should service tests extend a baseServiceTest?

Reply:
I'm going to do BaseIntegrationTest for now. I don't want to have services tested in a particular way. I think they're only interesting if we're testing the whole Spring show.